src/java/htsjdk/samtools/SAMTag.java

```diff
@@ -27,6 +27,47 @@
  * The standard tags for a SAM record that are defined in the SAM spec.
  */
 public enum SAMTag {
-    RG, LB, PU, PG, AS, SQ, MQ, NM, H0, H1, H2, UQ, PQ, NH, IH, HI, MD, CS, CQ, CM, R2, Q2, S2, CC, CP, SM, AM, MF,
-    E2, U2, OQ, FZ, SA, MC, BC
+    AM,
+    AS,
+    BC,
+    BQ,
+    CC,
+    CM,
+    CO,
+    CP,
+    CQ,
+    CS,
+    CT,
+    E2,
+    FI,
+    FS,
+    FZ,
+    LB,
+    H0,
+    H1,
+    H2,
+    HI,
+    IH,
+    MC,
+    MD,
+    MQ,
+    NH,
+    NM,
+    OQ,
+    OP,
+    OC,
+    PG,
+    PQ,
+    PT,
+    PU,
+    QT,
+    Q2,
+    R2,
+    RG,
+    RT,
+    SA,
+    SM,
+    TC,
+    U2,
+    UQ
```

Comment:
It is much easier to keep this updated via visual inspection with one tag per row.  Sorry for the verbosity.